trin-execution/src/era/manager.rs

```diff
@@ -30,10 +33,10 @@ pub struct EraManager {
 
 impl EraManager {
     pub async fn new(next_block_number: u64) -> anyhow::Result<Self> {
-        let http_client: Client = Config::new()
-            .add_header("Content-Type", "application/xml")
-            .expect("to be able to add header")
-            .try_into()?;
+        let mut headers = HeaderMap::new();
```

Comment:
Now that I think about it, it doesn't make sense for `CONTENT_TYPE` header to be set here and passed to `get_era_files` and `get_era1_files` (those functions are in a different crate, they control the URL and they are responsible for parsing the response).

So in this case, I would probably just create simple `Client::new()`, and each function would initialize headers that they need.

Not really part of this pr, but I think it's small enough to be included.